### PR TITLE
JS loading popup fix

### PR DESCRIPTION
**Pull request by lukaqueres**
Failed attempt for API fix

Loading popup is now closed on failed search.

### CODE REVIEW

File: src/DataFetcher.py [hunk 1]
```diff
@@ -229,9 +230,18 @@ def parse_yahoo_finance_analysis(self, response, *args, **kwargs):
   def fetch_yahoo_finance_quote(self):
     self.yahoo_finance_quote = YahooFinanceQuote(self.ticker_symbol)
     session = self._create_session()
+    # set_cookie = requests.get("https://fc.yahoo.com", allow_redirects=True)
+    # cookie = {"A3": "=d=AQABBDIjb2QCEP6Yfv8deeHPmCkMOJPr4LYFEgEBAQF0cGR5ZO-V7L8A_eMAAA&S=AQAAAtfWKlJiCqSO9jMVNukiJs8"}
+    # print(set_cookie.cookies)
+    # crumb = requests.get("https://query2.finance.yahoo.com/v1/test/getcrumb", cookies=set_cookie.cookies)
+    # print(crumb.headers)
+    # print(crumb.__dict__)
```
Comment (mrhappyasthma) [inline src/DataFetcher.py]: Can we remove all of these commented out lines?

Actually it looks like this file wasn't even really touched. Can we just revert it?

File: src/YahooFinance.py [hunk 2]
```diff
@@ -9,7 +9,7 @@ class YahooFinanceQuote:
   # Expects the ticker symbol as the only argument.
   # This can theoretically request multiple comma-separated symbols.
   # This could theoretically be trimmed down by using `fields=` parameter.
-  URL_TEMPLATE = 'https://query1.finance.yahoo.com/v6/finance/quote?symbols={}'
+  URL_TEMPLATE = 'https://query1.finance.yahoo.com/v7/finance/quote?symbols={}'
```
Comment (mrhappyasthma) [inline src/YahooFinance.py]: Seems unrelated to the PR. Can this be reverted?

Comment (lukaqueres) [reply src/YahooFinance.py]: Yes, I left my attempt, just in case, reversing it now.

File: src/DataFetcher.py (hunk 1 above)
Comment (lukaqueres) [reply src/DataFetcher.py]: Reversed

File: src/YahooFinance.py (hunk 2 above)
Comment (lukaqueres) [reply src/YahooFinance.py]: Done
